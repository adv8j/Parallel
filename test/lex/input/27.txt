int arr[5] ={1,2,3,4,5};
int arr2[2];
int arr3[5,10];
int n=15;
int arr4 =1..=n;
int arr4 =1..n;
int arr4 =1..==n; 
string s1="";
string s2="hello world!";

if(s2.length()==15)
{
    arr3[3]=7;
}

else if(s1.length()==3)
{
    arr4[7]=17;
}
else
{
    s1="Hey!";
}

for(int i=0;i<20;i+=1)
{
    arr[i]=i;

}

for(i in arr3)
{
    i+=1;
}

