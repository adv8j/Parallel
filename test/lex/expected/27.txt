INT
IDENTIFIER
LBRACKET
INT_LITERAL
RBRACKET
ASSIGN
LBRACE
INT_LITERAL
COMMA
INT_LITERAL
COMMA
INT_LITERAL
COMMA
INT_LITERAL
COMMA
INT_LITERAL
RBRACE
SEMICOLON
INT
IDENTIFIER
LBRACKET
INT_LITERAL
RBRACKET
SEMICOLON
INT
IDENTIFIER
LBRACKET
INT_LITERAL
COMMA
INT_LITERAL
RBRACKET
SEMICOLON
INT
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
INT
IDENTIFIER
ASSIGN
INT_LITERAL
RANGE_INCL
IDENTIFIER
SEMICOLON
INT
IDENTIFIER
ASSIGN
INT_LITERAL
RANGE
IDENTIFIER
SEMICOLON
INT
IDENTIFIER
ASSIGN
INT_LITERAL
RANGE_INCL
ASSIGN
IDENTIFIER
SEMICOLON
STRING
IDENTIFIER
ASSIGN
STRING_LITERAL
SEMICOLON
STRING
IDENTIFIER
ASSIGN
STRING_LITERAL
SEMICOLON
IF
LPAREN
IDENTIFIER
DOT
IDENTIFIER
LPAREN
RPAREN
EQ
INT_LITERAL
RPAREN
LBRACE
IDENTIFIER
LBRACKET
INT_LITERAL
RBRACKET
ASSIGN
INT_LITERAL
SEMICOLON
RBRACE
ELSE
IF
LPAREN
IDENTIFIER
DOT
IDENTIFIER
LPAREN
RPAREN
EQ
INT_LITERAL
RPAREN
LBRACE
IDENTIFIER
LBRACKET
INT_LITERAL
RBRACKET
ASSIGN
INT_LITERAL
SEMICOLON
RBRACE
ELSE
LBRACE
IDENTIFIER
ASSIGN
STRING_LITERAL
SEMICOLON
RBRACE
FOR
LPAREN
INT
IDENTIFIER
ASSIGN
INT_LITERAL
SEMICOLON
IDENTIFIER
LT
INT_LITERAL
SEMICOLON
IDENTIFIER
ADD_ASSIGN
INT_LITERAL
RPAREN
LBRACE
IDENTIFIER
LBRACKET
IDENTIFIER
RBRACKET
ASSIGN
IDENTIFIER
SEMICOLON
RBRACE
FOR
IDENTIFIER
IN
IDENTIFIER
LBRACE
IDENTIFIER
ADD_ASSIGN
INT_LITERAL
SEMICOLON
RBRACE
